index.html

```diff
@@ -55,7 +55,7 @@ <h1 class="section-title">I'm Anderson Glad to see you!</h1>
     </section>
     <section id="works-section">
       <div class="work-container">
-        <img class="align-item" src="./assets/images/work-1.png" alt="Tonic">
+        <img class="align-item card-image" src="./assets/images/work-1.png" alt="Tonic">
         <div class="work-aditional-info">
           <div class="work-info-container">
             <h1 class="align-item">Tonic</h1>
```

Comment:
- [ ] [OPTIONAL] We should maintain a single `h1` element per page for SEO purposes and a more logical structure for our page. I highly recommend you use an appropriate `hx` element such as `h2`, `h3` ... If you would like to have more information about the topic, kindly check this [article](https://www.semrush.com/blog/h1-tag/). 🧐️